report/src/main/webapp/app/layout/sidepanel/sidepanelToggle.directive.js

```diff
@@ -22,6 +22,8 @@ define(['angularAMD'], function (angularAMD) {
       ['$rootScope', 'localStorageService', ToggleSidepanelDirective]);
 
   function ToggleSidepanelDirective($rootScope, localStorageService) {
+    var $content = $('.main');
```

Comment:
This is also duplicated in the second directive, there should be a way of making that in single place